R/utils_memoised.R

```diff
@@ -1,24 +1,18 @@
-# taken from https://github.com/ropenscilabs/packagemetrics/blob/master/R/get_cran.R
+
 memoise_cran_db <- memoise::memoise({
   function() {
     cran <- tools::CRAN_package_db()
     # remove first instance of column name MD5Sum
-    cran <- cran[, -dplyr::first(which(names(cran) == "MD5sum"))]
-
-    # make it a tibble
-    cran <- dplyr::tbl_df(cran)
-
-    if (packageVersion("janitor") > "0.3.1") {
-      cran <- cran %>%
-        janitor::clean_names(case = "old_janitor") %>%
-        janitor::remove_empty("cols")
-    } else {
-      cran <- cran %>%
-        janitor::clean_names() %>%
-        janitor::remove_empty_cols()
-    }
+    cran <- cran[, - which(names(cran) == "MD5sum")[1]]
+
+    cran_name <- tolower(names(cran))
+    cran_name <- gsub("[-/@ ]", "_", cran_name)
+    cran_name <- gsub(".", "_", cran_name, fixed = TRUE)
+
+    names(cran) <- cran_name
```

Comment:
Instead of doing all of this preprocessing, I'd be totally fine having "messy" column names and the "MD5sum" column. I appreciate you going through and reproducing the functionality, but I think it might be better just to return the result of `tools::CRAN_package_db()` directly. 

This just reduces any separation from the raw data so that if changes get introduced to the result of the CRAN db we don't have to trace the issue through our preprocessing. 

```suggestion
```

I think there are only two lines that would be affected by this change.

#### `R/utils.R` (L#36):

```diff
- cran_packages <- crandb[["package"]]
+ cran_packages <- crandb[["Package"]]
```

#### `R/cran.R` (L#12):

```diff
- with(memoise_cran_db(), version[match(pkgname, package)])
+ with(memoise_cran_db(), version[match(pkgname, Package)])
```